components/spicedb/codegen/codegen.go

```diff
@@ -58,6 +58,7 @@ func GetCompiledSchema() *compiler.CompiledSchema {
 func GenerateDefinition(schema *compiler.CompiledSchema) string {
 
 	resource := "export type ResourceType ="
+	resourceTypes := "export const AllResourceTypes: ResourceType[] = ["
```

Comment:
I couldn't find a use for this. Is it a left over from an intermediate state?

Reply:
Might gotten stale after the rebase. Let me check.